pkg/scheduler/objects/application.go

```diff
@@ -1219,6 +1219,12 @@ func (sa *Application) tryPlaceholderAllocate(nodeIterator func() NodeIterator,
 	return allocResult
 }
 
+// check ask against both user headRoom and queue headRoom
+func (sa *Application) checkHeadRooms(ask *AllocationAsk, userHeadroom *resources.Resource, headRoom *resources.Resource) bool {
```

Comment:
Thanks @pbacsko 

The reason I introduced this method is because it need to be called at at least two places in  tryReservedAllocate(). I guess you did not see when you made the first comment here.  Just to confirm that you are ok with the change I made. Thanks.

Reply:
prefer to keep this as it is now.